lib/engine/game/base.rb

```diff
@@ -1353,15 +1354,15 @@ def train_help(_runnable_trains)
         []
       end
 
-      def queue_log!
+      def queue_log!(log_name)
```

Comment:
make this default nil so that you can queue_log without specify name and change the trivial references to use that